android/app/src/main/java/com/expensify/chat/utils/FileUtils.kt

```diff
@@ -0,0 +1,109 @@
+package com.expensify.chat.utils
+
+import android.content.Context
+import android.net.Uri
+import android.util.Log
+import android.webkit.MimeTypeMap
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+
+object FileUtils {
+    private const val tag = "FileUtils"
+    private const val directoryName = "Expensify"
+
+    private fun getInternalStorageDirectory(context: Context): File {
+        val internalStorageDirectory = File(context.filesDir.absolutePath, directoryName)
+        if (!internalStorageDirectory.exists()) {
+            internalStorageDirectory.mkdir()
+        }
+        return internalStorageDirectory
+    }
+
+    fun clearInternalStorageDirectory(context: Context) {
+        val internalStorageDirectory = File(context.filesDir.absolutePath, directoryName)
+        if (internalStorageDirectory.exists()) {
+            val files = internalStorageDirectory.listFiles()
+            for (file in files) {
+                file.delete()
+            }
```

Comment:
I think we should add a check if `files` exist, otherwise in case of some problems we may get a `NullPointerException`